server/src/main/java/org/opensearch/common/settings/IndexScopedSettings.java

```diff
@@ -191,6 +191,7 @@ public final class IndexScopedSettings extends AbstractScopedSettings {
                 BitsetFilterCache.INDEX_LOAD_RANDOM_ACCESS_FILTERS_EAGERLY_SETTING,
                 IndexModule.INDEX_STORE_TYPE_SETTING,
                 IndexModule.INDEX_STORE_PRE_LOAD_SETTING,
+                IndexModule.INDEX_TIERING_STATE,
```

Comment:
will remove this change from this pr and move it to the pr where this setting is introduced